### ISSUE TITLE
DAO Transfer

### ISSUE THREAD

**Issue by Torposoplos** [Jo2003/cd2netmd_gui#11]
Hi,

My devices is a Sony Hi-MD MZ-NH1 but DAO it's not available. Is it a bug? Its a TypeR/S device after all... Thanks.

**Comment by dogstyx** [comment]
Hi, I've tried the DAO option with my MDS-980, which presumably has the correct firmware, but the SP DAO option is greyed out, only the LP2 option is available. Am I missing something?

**Comment by dogstyx** [comment]
Right, I've now realised it was stated in the description but I missed that part, thanks!

This is off-topic, but I've noticed that the NetMD Wizard seems to always record the tracks in the "Artist - Track name" format, regardless of the actual name of the files. Is there any way to specify a different naming convention (for example, "Track name - Artist")?

**Comment by Jo2003** [comment]
There is no setting for this. If the track contains album / artist information it will be preferred over the file name. If there are no title - or track tags in the file, the file name will be used instead.

**Comment by dogstyx** [comment]
That makes sense, it would be great though being able to specify a different naming convention than "Artist - Track Name", even if it were though a config file. In my particular case I like to see first the track name that it is currently playing in my device display, as opposed to reading the artist name which doesn't help me much.

Then again everybody has a different experience, but here are my 2 cents. Thanks anyway for such a wonderful piece of software!
